src/transform/negate.js

```diff
@@ -2,8 +2,13 @@ var es = require("../es");
 
 function Negate(transform, node) {
     var expr = transform(node.expr);
-    var negate = es.Identifier(node.loc, "$negate");
-    return es.CallExpression(null, negate, [expr]);
+    if (typeof node.expr.data === "number") {
```

Comment:
This will work, but the correct check is `node.type === "Number"`. You never know what node type might have a numeric data field in the future, so please update the logic here.